src/msw/textctrl.cpp

```diff
@@ -608,6 +609,10 @@ bool wxTextCtrl::MSWCreateText(const wxString& value,
 #endif
         if ( !contextMenuConnected )
             Bind(wxEVT_CONTEXT_MENU, &wxTextCtrl::OnContextMenu, this);
+
+        // Zoom is not applied when this control is created in a wxFrame
+        // constructor, so delay applying it.
+        CallAfter(&wxTextCtrl::MSWSetInitialZoom);
```

Comment:
Not a big deal, but perhaps we should initialize `m_richDPIscale` to its real value here and only call `CallAfter()` if it's different from 1?

Reply:
I added a `== 1` check to `MSWSetRichZoom` so zoom is only set when it is required.